site/content/download.md

```diff
@@ -19,20 +19,6 @@ The Citra updater provides a easy interface to install, update and manage Citra.
  this is likely what you are looking for.
 <br />
 <br />
-
-<style>
- .alert {
-  padding: 20px;
-  background-color: #ff8e03;
-  color: white;
-  margin-bottom: 15px;
-}
- </style>
- 
-<div class="alert">
-  Notice: Citra does NOT support Apple silicon (M1/M2) MacOS devices. Our Mac builds may run through Rosetta, but you WILL encounter various issues that we won't provide support for. We may eventually support M1 Macs, but not at this time.
```

Comment:
> Now that Vulkan is added to Canary

What does Vulkan have to do with Rosetta?
Wouldn't it also need a Vulkan backend like MoltenVk now (in contrast to OpenGL - which is deprecated - but natively supported by macOS)?

Reply:
macOS builds were on hold because OpenGL support was dropped, now they are back as the Vulkan PR supports MoltenVK. In the meantime support for native ARM64 was also added, it just was never in a release build due to the aforementioned block.